packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -63,6 +63,11 @@ contract OptimismPortal is Initializable, ResourceMetering, Semver {
      */
     mapping(bytes32 => bool) public finalizedWithdrawals;
 
+    /**
+     * @notice Reserve extra slots (to a total of 50) in the storage layout for future upgrades.
+     */
+    uint256[48] private __gap;
```

Comment:
I previously took this out, I don't think we need them in our concrete contracts which are not inherited by anything in production. If we put this one here, then we should have one in the `L2OutputOracle` too.

It's fine either way, but a bit cleaner not to IMO.

Reply:
Ok i'm down to remove it

Reply:
I've addressed your comment @maurelian